tests/cases/standalone/common/function/expression.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE t(a INTEGER, ts timestamp time index);
+
+INSERT INTO t VALUES (1, 1);
+
+INSERT INTO t VALUES (null, 2);
+
+INSERT INTO t VALUES (3, 3);
```

Comment:
nit-picking: merge them into one `INSERT` clause

Reply:
I'll adjust it now